examples/image_reconstruction.ipynb

```diff
@@ -0,0 +1,705 @@
+   },
+   "outputs": [],
+   "source": [
+    "# specifying loss and optimization function\n",
+    "\n",
+    "criterion = nn.MSELoss()\n",
+    "\n",
+    "optimizer = torch.optim.Adam(network.parameters(), lr=learning_rate)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "pPRZ9igMw_P3",
+    "outputId": "cf3107fc-17d9-404d-b606-64893f13e2fa"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Epoch: 1/5 Step: 1500/1500 100.00% |████████████████████|22.35s loss: 0.042889 val_loss: 0.029158\n",
+      "Epoch 1: val_loss improved from inf to 0.02916, saving file to saves/best_weight.ckpt\n",
+      "Epoch: 2/5 Step: 1500/1500 100.00% |████████████████████|22.10s loss: 0.026634 val_loss: 0.023818\n",
+      "Epoch 2: val_loss improved from 0.02916 to 0.02382, saving file to saves/best_weight.ckpt\n",
+      "Epoch: 3/5 Step: 1500/1500 100.00% |████████████████████|22.99s loss: 0.022496 val_loss: 0.021518\n",
+      "Epoch 3: val_loss improved from 0.02382 to 0.02152, saving file to saves/best_weight.ckpt\n",
+      "Epoch: 4/5 Step: 1500/1500 100.00% |████████████████████|21.96s loss: 0.020868 val_loss: 0.020440\n",
+      "Epoch 4: val_loss improved from 0.02152 to 0.02044, saving file to saves/best_weight.ckpt\n",
+      "Epoch: 5/5 Step: 1500/1500 100.00% |████████████████████|21.83s loss: 0.020034 val_loss: 0.019797\n",
+      "Epoch 5: val_loss improved from 0.02044 to 0.01980, saving file to saves/best_weight.ckpt\n",
+      "Restoring model from saves/best_weight.ckpt\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[{'epoch': 1,\n",
+       "  'loss': 0.042888817086815835,\n",
+       "  'time': 22.35295373400004,\n",
+       "  'val_loss': 0.029158337180813152},\n",
+       " {'epoch': 2,\n",
+       "  'loss': 0.02663403686011831,\n",
+       "  'time': 22.10114606299976,\n",
+       "  'val_loss': 0.023817934319376944},\n",
+       " {'epoch': 3,\n",
+       "  'loss': 0.02249614226445556,\n",
+       "  'time': 22.987112360000083,\n",
+       "  'val_loss': 0.02151769397656123},\n",
+       " {'epoch': 4,\n",
+       "  'loss': 0.020868006257961192,\n",
+       "  'time': 21.95595156699983,\n",
+       "  'val_loss': 0.020440032189091045},\n",
+       " {'epoch': 5,\n",
+       "  'loss': 0.02003355091251433,\n",
+       "  'time': 21.834973953999906,\n",
+       "  'val_loss': 0.019796804750959077}]"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {
+      "tags": []
+     },
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Poutyne Model on GPU\n",
+    "model = Model(network, optimizer, criterion, device=device)\n",
+    "\n",
+    "# Train\n",
+    "model.fit_generator(trn_dataloader, val_dataloader, epochs=num_epochs, callbacks=callbacks, progress_options={'coloring': False})"
```

Comment:
```suggestion
    "model.fit_generator(train_dataloader, valid_dataloader, epochs=num_epochs, callbacks=callbacks)"
```
Following above suggestions. Also, I don't see why we would disable coloring here.